fadecandy_driver/src/ros/fadecandy_driver_ros.h

```diff
@@ -0,0 +1,50 @@
+//
+// Copyright (c) 2021 Eurotec
+//
+
+#include "../fadecandy_driver.h"
+#include <diagnostic_updater/diagnostic_updater.h>
+#include <fadecandy_msgs/LEDArray.h>
+#include <ros/publisher.h>
+#include <ros/subscriber.h>
+
+namespace fadecandy_driver
+{
+class FadecandyDriverRos : public FadecandyDriver
+{
+  //!
+  //! \brief The FadecandyDriverRos class wraps ROS
+  //!
+  //!
+public:
+  FadecandyDriverRos();
+  void run();
+  bool initialized_;
```

Comment:
documenting public interfaces is more important than private ones but at least be consistent